### PR TITLE
Add selected models testing in CI for TF-OVIE-CPU

### PR DESCRIPTION
**Pull request by bani-intelaipg**
BuildKite CI log at https://buildkite.com/ngraph/ngtf-ie-modeltest/builds/22

### CODE REVIEW

File: test/model_level_tests/models/TF_OV/run_infer_single.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+IMAGE=$2
+if [ "${IMAGE}" == "" ]; then
+    echo "Error: image not specified!" && exit 1
+elif [[ ! ${IMAGE} =~ *.* ]]; then
+    IMAGE="${IMAGE}.jpg"
+fi
+INFER_PATTERN=$3
+if [ "${INFER_PATTERN}" == "" ]; then
+    echo "Error: expected pattern not specified!" && exit 1
+fi
```
Comment (adk9) [inline test/model_level_tests/models/TF_OV/run_infer_single.sh:21]: Can we use a default for this if one isn't specified?

Comment (bani-intelaipg) [reply test/model_level_tests/models/TF_OV/run_infer_single.sh:21]: I thought about it, but I think it's best for the developer to use args/params as needed. There is an example at top of the file.

File: test/model_level_tests/models/TF_OV/models_cpu.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+resnet_50_v2  bike  'mountain bike, all-terrain bike'
+resnet_v1_50  bike  'mountain bike, all-terrain bike'
+resnet_v2_101  bike  'mountain bike, all-terrain bike'
+resnet_v2_152  bike  'mountain bike, all-terrain bike'
+resnet_v2_50  bike  'mountain bike, all-terrain bike'
+inception_v3  bike  'mountain bike, all-terrain bike'
+inception_v4  bike  'mountain bike, all-terrain bike'
+
+#vgg_16  bike  'mountain bike, all-terrain bike, off-roader\s*( 11.'
+#vgg_19  bike  'tricycle, trike, velocipede\s*( 10.'
```
Comment (kanvi-nervana) [inline test/model_level_tests/models/TF_OV/models_cpu.txt:16]: Why are these commented?

Comment (bani-intelaipg) [reply test/model_level_tests/models/TF_OV/models_cpu.txt:16]: Will activate as soon as Mustafa fixes the script name bug in models repo

**Review by adk9 — APPROVED** [review]

**Pull request merged**